tests/hmm/hmumu_utils.py

```diff
@@ -1546,7 +1559,7 @@ def get_selected_jets_id(
     )
     if dataset_era == "2017":
         jet_eta_pass_veto = NUMPY_LIB.logical_or(
-            (raw_pt > jet_veto_raw_pt),
+            (jet_puid == "tight"), #(raw_pt > jet_veto_raw_pt),
```

Comment:
I also want to draw your attention to this line. This is to add the new PU ID training for jets in eta [2.6,3.1] . Instead of vetoing those jets now, we just apply tight ID. I don't know if the new PU ID is included for 2017 in the nanoAOD v6

Reply:
the new PU ID is not included for 2017 in nanoAOD v6 I think